pkg/network/setup/netstat.go

```diff
@@ -104,7 +105,13 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		}
 	}
 
+	primaryInterfaceStatus, interfacesStatus := netvmispec.PopInterfaceByNetwork(interfacesStatus, netvmispec.LookupPodNetwork(vmi.Spec.Networks))
```

Comment:
Maybe it would be easier to add the `default` attribute to status interfaces and use the query language.